tests/test_transaction.py

```diff
@@ -37,6 +37,23 @@ def test_sign_transaction(ethereum_accounts):
     assert signed.signature.hex() == expected
 
 
+def test_sign_customtransaction_data_body(ethereum_accounts):
```

Comment:
Can you add another test to make sure we can go from `SignedTransaction` to `CustomTransaction` via `.transaction()`?

Reply:
Sure.

Reply:
Done on d369251287d32bbff86aa9bda9d50ec5a166f6ad

Good call! There was a bug on the signed.transaction() conversion.